server/src/main/java/org/elasticsearch/index/get/GetResult.java

```diff
@@ -397,6 +445,8 @@ public boolean equals(Object o) {
         }
         GetResult getResult = (GetResult) o;
         return version == getResult.version &&
+                seqNo == getResult.seqNo &&
+                primaryTerm == getResult.primaryTerm &&
```

Comment:
We should also add seqno and term to hashCode.

Reply:
+1